### ISSUE TITLE
Subscription Page has some strange issues

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#6410]
OK this is a strange one and I will try to explain as best I can. This subscription link will only show up if the word "Subscribed" is written inside the post. We discovered that accidentally when suddenly the menu appeared. It took us time to figure it out, but when you delete that word "Subscribed" the menu disappears.

![Screen Shot 5782-07-18 at 20 43 13](https://user-images.githubusercontent.com/58882903/159342616-4368a801-3320-4b12-b1a3-b1eda436b31d.jpg)

Here is how we add the word to a post:
![Screen Shot 5782-07-18 at 20 48 18](https://user-images.githubusercontent.com/58882903/159343120-9ff66dc7-71b8-47bb-8a4d-a045c36cd735.jpg)

This is strange because once I subscribe to any channel that menu link should appear and because none of our posts had the word in them than that menu item doesn't show to any user.

Also once you open the subscribed page and view the list of channels, the channel name is missing its URL to the channel, this can be very helpful for us if we want to build menus and use those URLS.
![Screen Shot 5782-07-18 at 20 44 21](https://user-images.githubusercontent.com/58882903/159343707-11579d79-20ca-4450-8a95-420afe9d17ec.jpg)

So in short, something is responding to that keyword "Subscribed" in posts and only then will the menu appear.

**Comment by DanielnetoDotCom** [comment]
Can you please record a video so it will help me understand?

**Comment by guymass** [comment]
Sure.

**Comment by guymass** [comment]
Was waiting for my friend to provide me with the video but he just cant get to it yet, and I also am waiting to see what it is really so for now sorry I still don't have a video to show it, maybe in few days.
